src/main/java/examples/PlotAScatter.java

```diff
@@ -0,0 +1,30 @@
+package examples;
+
+import dataloader.CsvDataLoader;
+import tech.tablesaw.api.Table;
+import visualizations.ScatterChart;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Category: Plotting
+ * ID: PlotABar
+ * Description: Load a CSV file and plot a scatter chart with 3 numeric variables
+ */
+public class PlotAScatter {
+
+    public static void main(String[] args) throws IOException {
+        File file = new File("test_data/humans_data.csv");
```

Comment:
Let's keep the ```test_data/``` only for the tests and use ```data/``` for the examples???

Reply:
updated file path for all cases